net-misc/i2pd/i2pd-2.6.0-r3.ebuild

```diff
@@ -53,27 +53,38 @@ src_configure() {
 
 src_install() {
 	cmake-utils_src_install
+
+	# config
+	insinto "/etc/i2pd"
+	doins "${S}/docs/i2pd.conf"
+	doins "${S}/debian/subscriptions.txt"
+	doins "${S}/debian/tunnels.conf"
```

Comment:
docs/i2pd.conf is a sample config file. We place it in /etc/i2pd as a starting config for user to edit. This file used to be in a different place upstream, moved to docs recently.